<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetalhesViagemWidget</class>
 <widget class="QWidget" name="DetalhesViagemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="detalhesWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>551</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblDetalheFotoDisplay">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>209</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Sem Imagem</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>510</y>
      <width>195</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="btnFotoAnterior">
       <property name="text">
        <string>&lt; Anterior</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFotoProxima">
       <property name="text">
        <string>Próxima &gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>3</y>
      <width>521</width>
      <height>207</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="lblDetalheNome">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Selecione uma Viagem</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblDetalheDestino">
           <property name="text">
            <string>Destino:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblDetalheData">
           <property name="text">
            <string>Data:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="lblDescricaoTitulo">
           <property name="text">
            <string>Descrição:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="browserDetalheDescricao"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblFotosTitulo">
       <property name="text">
        <string>Fotos:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
